### PR TITLE
Check if the promoted images are built from the latest commit

### PR DESCRIPTION
**Pull request by hongkailiu**
/hold

wait for rehearsal ...
requires https://github.com/openshift/ci-tools/pull/2235/files

### CODE REVIEW

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -1982,25 +1982,38 @@ periodics:
...
+      - --github-endpoint=https://api.github.com
+      - --github-graphql-endpoint=http://ghproxy/graphql
       - --ignored-image-stream-tags
       - ^ocp\S*/\S+:machine-os-content$
       - --ignored-image-stream-tags
       - '^openshift/origin-v3.11:'
       - --ignored-image-stream-tags
       - '^origin/4.([6-9]|\d\d+):'
-      - --dry-run=false
+      - --dry-run=true
```
Comment (hongkailiu) [inline ci-operator/jobs/infra-periodics.yaml:2004]: Will change it back after passing the rehearsal.

File: ci-operator/jobs/infra-periodics.yaml [hunk 2]
```diff
@@ -1982,25 +1982,38 @@ periodics:
     org: openshift
     repo: release
   labels:
-    ci.openshift.io/role: infra
+    #ci.openshift.io/role: infra
```
Comment (hongkailiu) [inline ci-operator/jobs/infra-periodics.yaml:1985]: Will change it back after passing the rehearsal.

**Comment by stevekuznetsov** [comment]
```go
fmt.Sprintf("DPTP-1234")
```

**Comment by stevekuznetsov** [comment]
fmt.Sprintf("DPTP-1234")

**Comment by stevekuznetsov** [comment]
DPTP-1234

**Comment by stevekuznetsov** [comment]
```
DPTP-1234
```

**Comment by hongkailiu** [comment]
https://issues.redhat.com/browse/DPTP-1899

**Pull request closed without merging**